specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/examples/JavaComponents_List.json

```diff
@@ -48,10 +48,40 @@
                 "fqdn": "myjavacomponent.myenvironment.test.net"
               }
             }
+          },
+          {
+            "id": "/subscriptions/8efdecc5-919e-44eb-b179-915dca89ebf9/resourceGroups/examplerg/providers/Microsoft.App/managedEnvironments/myenvironment/javaComponents/reddog",
+            "name": "reddog",
+            "type": "Microsoft.App/managedEnvironments/javaComponents",
+            "properties": {
+              "componentType": "SpringCloudGateway",
+              "provisioningState": "Succeeded",
+              "configurations": [
+                {
+                  "propertyName": "spring.cloud.gateway.enabled",
+                  "value": "true"
+                }
+              ],
+              "springCloudGatewayRoutes": [
+                {
+                  "id": "route1",
+                  "uri": "https://httpbin.org",
```

Comment:
I'd like to suggest not linking random non-MS websites from our examples

Reply:
Of, course. I will change this.